raiden/tests/fixtures/api.py

```diff
@@ -15,11 +15,15 @@
 from raiden.network.discovery import Discovery
 from raiden.tests.utils.apitestcontext import ApiTestContext
 
+_SELF = object()
 
-def wait_for_listening_port(port_number, tries=10, sleep=0.1):
+
+def wait_for_listening_port(port_number, tries=10, sleep=0.1, pid=_SELF):
+    if pid is _SELF:
```

Comment:
Can't `None` be used instead of a new global object?

Reply:
Yes, I think it can. Not sure anymore why I went with the sentinel object.